data/StevenBlack/hosts

```diff
@@ -2767,4 +2765,4 @@
 0.0.0.0 trilns.com
 0.0.0.0 piasuna.gdn
 0.0.0.0 tdreg.icu
-0.0.0.0 tdreg.top
\ No newline at end of file
+0.0.0.0 tdreg.top
```

Comment:
Any concerns about this newline? :grimacing:

Reply:
No concerns at all 😄